engineapi/instance_manager.go

```diff
@@ -240,10 +240,22 @@ func parseProcess(p *imapi.Process) *longhorn.InstanceProcess {
 			Conditions: p.ProcessStatus.Conditions,
 			PortStart:  p.ProcessStatus.PortStart,
 			PortEnd:    p.ProcessStatus.PortEnd,
+
+			// These fields are not used, maybe we can deprecate them later.
```

Comment:
Sounds good. 

// FIXME These fields are not used, maybe we can deprecate them later.

Reply:
Updated. Thank you!